src/intrprtr.c

```diff
@@ -379,22 +328,21 @@ void IntrAbortCoding(Obj lvars)
 **  arguments. <options> is 1 if options were present after the ':' in which
 **  case the options have been read already.
 */
-void            IntrFuncCallBegin ( void )
+void IntrFuncCallBegin(IntrState * intr)
 {
     /* ignore or code                                                      */
     SKIP_IF_RETURNING();
     SKIP_IF_IGNORING();
-    if ( STATE(IntrCoding)    > 0 ) { CodeFuncCallBegin(); return; }
-
+    if (intr->coding > 0) {
+        CodeFuncCallBegin();
+        return;
+    }
```

Comment:
I accidentally clang-formatted this `if`. But that makes me wonder: since I am touching all these `if (IntrCoding)` etc. checks anyway, is there any harm in running `git clang-format` on them? Doesn't make the `git blame` experience worse here, does it? Just reviewing would be slightly more annoying, but that could be alleviated by putting those reformatting changes into a second commit.

Reply:
Maybe add an extra commit which does the formatting, just so it will be easy to spot if looking through history/blame?

Reply:
The problem in general reformatting is that it can make it slightly harder to see which bits of the code have changed (although in this case, it doesn't really make any difference, as you say)

Reply:
I still think it gets slightly more annoying because the diff is bigger; and when a single line is just modified, you can see easily in the diff what was inserted/removed; while if I reformat it into multiple lines, you can't see that quite as easily; it might not even be easy to match old and new code. Anyway, it shouldn't be *that* bad; I'll put it into a second commit, so we can also drop it if we change our minds.